scripts/Makefile

```diff
@@ -0,0 +1,96 @@
+# to install geocint just run 
+# make install configuration_file=/path/to/your/config/file/config.inc.sh
+
+# Add here export for every varible from configuration file that you are going to use in targets
+export PGDATABASE = $(shell sed -n -e '/^PGDATABASE/p' ${configuration_file} | cut -d "=" -f 2)
+export PGUSER = $(shell sed -n -e '/^PGUSER/p' ${configuration_file} | cut -d "=" -f 2)
+export GENERAL_FOLDER = $(shell sed -n -e '/^GENERAL_FOLDER/p' ${configuration_file} | cut -d "=" -f 2)
+export SLACK_CHANNEL = $(shell sed -n -e '/^SLACK_CHANNEL/p' ${configuration_file} | cut -d "=" -f 2)
+export SLACK_BOT_NAME = $(shell sed -n -e '/^SLACK_BOT_NAME/p' ${configuration_file} | cut -d "=" -f 2)
+export SLACK_BOT_EMOJI = $(shell sed -n -e '/^SLACK_BOT_EMOJI/p' ${configuration_file} | cut -d "=" -f 2)
+export SLACK_BOT_KEY = $(shell sed -n -e '/^SLACK_BOT_KEY/p' ${configuration_file} | cut -d "=" -f 2)
+
+export GEOCINT_WORK_DIRECTORY = $(shell sed -n -e '/^GEOCINT_WORK_DIRECTORY/p' ${configuration_file} | cut -d "=" -f 2)
+export PRIVATE_REPO_NAME = $(shell sed -n -e '/^PRIVATE_REPO_NAME/p' ${configuration_file} | cut -d "=" -f 2)
```

Comment:
I find `PRIVATE_REPO_NAME` to be very confusing name for what is it used. does that thing we specifying here need to be repository? why? why private? will something go wrong if it will be public? what is repo?

Reply:
What variable name would you prefer, any options?

Reply:
replaced with CUSTOM_PART_FOLDER_NAME